src/features/blog/ContentEdit/content-edit.tsx

```diff
@@ -276,16 +277,14 @@ export const ContentEdit = (props: ContentEditProps) => {
   }, [formik.values]);
 
   const savingIndicatorElement = (
```

Comment:
Can we have this savingIndicatorElement as a common component to reuse in other modules ?

Reply:
Done